Python/pywarpx/WarpX.py

```diff
@@ -107,6 +107,64 @@ def get_bucket(self, bucket_name):
             self._bucket_dict[bucket_name] = bucket
             return bucket
 
+    @staticmethod
+    def read_dims_from_file(filename, visited=None):
```

Comment:
There is already Tools/Parser/input_file_parser.py. Could this be reused? (Though would have to be updated to handle FILE.)

Reply:
Oh thanks, I was not aware of this! :)

Reply:
To be improved separately, as discussed in the developer meeting today.
Once we make this a `warpx-tools` package and auto-install it as a sub-package of `pywarpx` we can use it here! :)